addons/sale/models/sale_config_settings.py

```diff
@@ -159,3 +155,7 @@ def _onchange_sale_tax(self):
                 'group_show_price_total': True,
                 'group_show_price_subtotal': False,
             })
+
+    @api.depends('group_show_price_total')
+    def _compute_sale_show_tax(self):
```

Comment:
Coding guidelines note: we generally put compute before onchanges, in the same order as computed fields are declared.